example/example/spiders/books_06.py

```diff
@@ -12,34 +12,34 @@
 import scrapy
 import attr
 
-from scrapy_po import WebPage, ItemPage
+from scrapy_po import WebPage, ItemWebPage, Injectable
 
 
-class ListingsPiece(WebPage):
+class ListingsExtractor(WebPage):
```

Comment:
We would need some generic posfix to use in these classes that only deals with extraction from a part of a page. I used here `Extractor`.

Reply:
`Extractor` sounds fine.